docs/source/demo.rst

```diff
@@ -95,7 +95,7 @@ equivalent command using:
 .. code-block:: bash
 
     $ source ga4gh-env/bin/activate
-    (ga4gh-env) $ ga4gh_client readgroupsets-search --datasetIds XXXX http://localhost:8000/current
+    (ga4gh-env) $ ga4gh_client readgroupsets-search --dataset_ids XXXX http://localhost:8000/current
```

Comment:
I think these command line arguments might be hyphenated now.  I think argparse might change these on the fly. Can you check if this works now? I don't have a strong opinion about which way it should be for CLI stuff, but I'd like to check if these examples still work.